packages/terra-brand-footer/src/terra-dev-site/doc/brand-footer/UpgradeGuide.2.doc.mdx

```diff
@@ -3,6 +3,47 @@ import { Badge } from 'terra-brand-footer/package.json?dev-site-package';
 <Badge />
 
 # Terra Brand Footer Upgrade Guide
+## Upgrading from v2 to v3
+### Changes from version 2 to version 3
+#### Removed props
+* Removed `links`
+
+### Uplift to V3
+```diff
+import React from 'react';
+import BrandFooter from '../../../BrandFooter';
+
+class Default extends React.Component {
+  constructor() {
+    super();
+    this.handleFilterDate = this.handleFilterDate.bind(this);
+  }
+
+  render() {
+      return (
+        <BrandFooter
+-        links={[
+-            { text: 'Terra UI', href: 'http://terra-ui.com/static/#/site/home' },
+-            { text: 'Cerner', href: 'https://www.cerner.com/', target: '_blank' },
+-        ]}
++        sections={[
++            {
++            headerText: 'Terra UI Links',
++            links: [
++                { text: 'Terra UI', href: 'http://terra-ui.com/static/#/site/home' },
++                { text: 'Cerner', href: 'https://www.cerner.com/', target: '_blank' },
++            ],
++           },
++        ]}
+        />
+    );
+  }
+};
+```
+### Changes from version 2 to version 3
+#### Removed props
+* Removed `links`
+
```

Comment:
Oops this was added 2x

Reply:
Updated here https://github.com/cerner/terra-framework/pull/1167/commits/b7d34c6ed6202c60788979859d48f35354d4d505.